### DISCUSSION TITLE
Using the same domain on Vercel and AWS

### DISCUSSION THREAD

**Discussion by enricoschaaf** [vercel/vercel#4753] [Help] [answered]
Is it possible to use the same domain for your frontend hosted on Vercel and your backend running on AWS. For example your frontend is example.com and your API is accessed under example.com/api/*.

**Comment by mcsdevv** [comment, marked as answer]
Hello @enricoschaaf, this is certainly possible, you can follow the guidance here to do so: https://vercel.com/knowledge/pointing-subdomains-to-external-services
